tests/gtest/test_rcp_host_api.cpp

```diff
@@ -64,40 +64,41 @@ static void MainloopProcessUntil(otbr::MainloopContext    &aMainloop,
 
 TEST(RcpHostApi, DeviceRoleChangesCorrectlyAfterSetThreadEnabled)
```

Comment:
may also add cases that enable after enable Thread remains enabled; disable after disable Thread remains disabled?

Reply:
Added "enable after enable" in step 3 and "disable after disable" in step 5.